### PR TITLE
task/COOKS-201: separate dashboard code.

### PR DESCRIPTION
**Pull request by jarosenb**
This PR adds a separate React module for serving the Cooks dashboard. 

Local dev instructions:
- Start up the containers with docker-compose
- Start up the dashboard frontend with `npm run dev` in the `protx-client` folder.
- In Django CMS, make 3 pages with iframe snippets that have the following markup:

```
<p><span style="color: #ffffff;"><iframe frameborder="0" height="1600" width="100%" src="https://cep.dev/protx/dash/demographics"></iframe></span></p>
```
```
<p><span style="color: #ffffff;"><iframe frameborder="0" height="1600" width="100%" src="https://cep.dev/protx/dash/maltreatment"></iframe></span></p>
```
```
<p><span style="color: #ffffff;"><iframe frameborder="0" height="1600" width="100%" src="https://cep.dev/protx/dash/analytics"></iframe></span></p>
```

Navigate to those pages and check that the dashboard loads.

### CODE REVIEW

**Review by nathanfranklin — APPROVED** [review]
Looks great 💯 

----------

left some comments that be addressed here or later before we get to main.

**Review by taoteg — APPROVED** [review]
Fantastic!

File: protx/app.py [hunk 1]
```diff
@@ -11,4 +11,10 @@
 
 api.add_namespace(protx_api, path="/protx/api")
 
+
+@app.route("/protx/dash/<path:path>")
+def index(path=None):
+    # TODO: Render alternate template for the compiled bundle.
```
[the remark this answers is not in the document]
Comment (nathanfranklin) [reply protx/app.py:17]: adding todo in https://github.com/TACC/protx-dashboard/pull/1

File: protx-client/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react'
+import ProTx from './components/ProTx';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+
+function App() {
+  //TODO: redirect if user hasn't passed onboarding.
```
[the remark this answers is not in the document]
Comment (nathanfranklin) [reply protx-client/src/App.jsx:6]: adding todo to create ticket in #1

**Pull request merged**